V5 Release note - Quality part (#873)
akeneo/pim-helpcenter: content/updates/quality/1-The-quality-score.md

> # Introducing our Quality score!

Reviewer: ```suggestion
# Introducing our quality score!
```

> ## A new way of scoring...

Reviewer: ```suggestion
## A new way of scoring
```

Author: See my comment below.

::: meta-data type="New" ee-only="false" link-to-doc="../articles/understand-data-quality.html#about-data-quality-insights"
:::
Measuring your product quality is now easier than ever thanks to the **quality score**, our new simplified way to score your product. It replaces the enrichment and consistency grades for better clarity.

![The new quality score](../img/quality-score.png)

You can now find this score next to the completeness in the header of each product page, as well as in the product grid. Don't hesitate to use it as a filter to focus your attention on the products with low-quality scoring.

![The new quality score in the product edit form](../img/quality-score-in-pef.png)

> ## ...All your products...

Reviewer: I don't understand the ... in every title

Author: The three titles form a single title. "A new way of scoring / All your products / On selected attributes!"

Reviewer: ah! ok :-)

::: meta-data type="New" ee-only="false"
:::

> You'll also notice that this score and the Data Quality Insights tab are now available for both your variant products and product models! You will get specific quality scores for all your variant products so you can quickly identify the list of attributes you need to improve for a specific variant product. We also deliver a quality score at the product model level so you can see at a glance on which level the attributes need to be improved (at the variant product or at the product model level).

Reviewer: ```suggestion
You'll also notice that this score and the Data Quality Insights tab are now available for both your variant products and product models! You get specific quality scores for all your variant products so you can quickly identify the list of attributes you need to improve for a specific variant product. We also deliver a quality score at the product model level so you can see at a glance on which level the attributes need to be improved (at the variant product or at the product model level).
```

![DQI on variant products](../img/dqi-for-a-variant-product.png)

> When you have improvement recommendations, you can simply click on your attribute to go directly to it and improve it under the Data Quality Insights tab. This will make your job easier and allow you to react to poor data quality faster.

Reviewer: ```suggestion
When you have improvement recommendations, you can simply click on your attribute and improve it right from the Data Quality Insights tab. This will make your job easier and allow you to react to poor data quality faster.
```

## ...On selected attributes!
::: meta-data type="New" ee-only="false" link-to-doc="../articles/manage-data-quality.html#data-quality-insights-activation-and-deactivation"
:::

> You can now select the product attributes to take into account when calculating your product quality score. You can toggle this option at the attribute group level under the attribute group settings page.

Reviewer: ```suggestion
You can now select the product attributes you want to take into account when calculating your product quality score. You can enable this option at the attribute group level under the attribute group settings page.
```

Reviewer: What do you mean by 'under the attribute group' -> shouldn't it be ON the (...) page?

Author: On each attribute group page, you have a toggle button to choose whether you want it to be part of the quality score calculation.

Reviewer: ok, so I guess it's on, under would mean it's physically under another component

Author: ```suggestion
You can now select the product attributes you want to take into account when calculating your product quality score. You can enable this option at the attribute group level on the attribute group settings page.
```

:::
Measuring your product quality is now easier than ever thanks to the **quality score**, our new simplified way to score your product. It replaces the enrichment and consistency grades for an enhanced clarity.

![The new quality score](../img/quality-score-in-pef.png)

You can now find this score next to the completeness in the header of each product page, as well as in the product grid. Don't hesitate to use it as a filter to focus your attention on the products with low-quality scoring.

## ...All your products...
::: meta-data type="New" ee-only="false"
:::
You'll also notice that this score and the Data Quality Insights tab are now available for both your variant products and product models! Get specific quality scores for all your product variants and quickly identify the list of attributes you need to improve for a specific variant product. We also deliver a quality score at the product model level in which you will see at a glance on which level the attribute to improved is (product variant or model level).

![Data Quality Insights on variant products](../img/dqi_variant_products.png)

> To enhance your productivity, you can also notice that, under the Data Quality Insights tab, if you have any improvement recommendations, you can simply click on your attribute to directly access it and improve it! 

Author: ```suggestion
When you have improvement recommendations, you can simply click on your attribute to go directly to it and improve it under the Data Quality Insights tab. This will make your job easier and allow you to react to poor data quality faster.
```

![Ease data quality improvements in the PEF](../img/ease_data_quality_improvements_in_pef.png)

## ...On some given attributes!
::: meta-data type="New" ee-only="false" link-to-doc="../articles/manage-data-quality.html#data-quality-insights-activation-and-deactivation"
:::

> You can now fine-tune which product attributes you want to count in your quality score, by enabling/disabling the feature at the attribute group level. You can toggle this under the attribute group setting page. 😉

Author: ```suggestion
You can now select the product attributes to take into account when calculating your product quality score. You can toggle this option at the attribute group level under the attribute group settings page.
```
